.github/workflows/build-engine-container.yml

```
@@ -110,10 +137,18 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: <Setup> Login to DockerHub
+      - name: <Setup> Login to GitHub Container Registry (ghcr.io)
```

（細かいけど）

まあ`registry`に書いてるしかっこ内はなくてもなんとかわかりそう！
（逆に分からないならここじゃなくて別のところで書くのが良さそう。）
```suggestion
      - name: <Setup> Login to GitHub Container Registry
```

---

いいと思います。削除します。